src/pages/Punch/VerifyPunch/VerifyPunch.tsx

```diff
@@ -34,204 +30,187 @@ const ButtonGroup = styled.div`
-    //                                     )
-    //                                 )
-    //                             }
-    //                         >
-    //                             Unclear
-    //                         </Button>
-    //                         <Button
-    //                             disabled={
-    //                                 punchActionStatus === AsyncStatus.LOADING
-    //                             }
-    //                             onClick={(): Promise<void> =>
-    //                                 handlePunchAction(PunchAction.REJECT, () =>
-    //                                     history.push(
-    //                                         removeSubdirectories(url, 1) +
-    //                                             '/clear'
-    //                                     )
-    //                                 )
-    //                             }
-    //                         >
-    //                             Reject
-    //                         </Button>
+                    <Button
+                        disabled={punchActionStatus === AsyncStatus.LOADING}
+                        onClick={(): Promise<void> =>
+                            handlePunchAction(PunchAction.VERIFY, () => {
+                                history.push(
+                                    removeSubdirectories(url, 2) + '/punch-list'
+                                );
+                            })
+                        }
+                    >
+                        Verify
+                    </Button>
+                </>
+            );
+        }
+    };
 
-    //                         <Button
-    //                             disabled={
-    //                                 punchActionStatus === AsyncStatus.LOADING
-    //                             }
-    //                             onClick={(): Promise<void> =>
-    //                                 handlePunchAction(
-    //                                     PunchAction.VERIFY,
-    //                                     () => {
-    //                                         history.push(
-    //                                             removeSubdirectories(url, 2)
-    //                                         );
-    //                                     }
-    //                                 )
-    //                             }
-    //                         >
-    //                             Verify
-    //                         </Button>
-    //                     </ButtonGroup>
-    //                 </VerifyPunchWrapper>
-    //             </>
-    //         );
-    //     } else if (fetchPunchItemStatus === AsyncStatus.ERROR) {
-    //         return <ErrorPage title="Unable to load punch item." />;
-    //     } else {
-    //         return <SkeletonLoadingPage text="Loading punch item" />;
-    //     }
-    // };
+    return (
+        <>
+            {punchItem ? (
```

Comment:
Is this needed? Punch is determined to be != undefined in PunchPage before rendering this component.